saleor/saleor-dashboard
# [Feature Flags] GraphQL build multiple schemas

I want to merge this change because it allow us to generated multiple schemas base on feature flags



**PR intended to be tested with API branch:** 

### How to test
1. Prepare a new introspection file, you can use the fetch-schema script, the important introspection file should contain a suffix e.g. introspection.test.json
2. Add somewhere new queries.test.ts file, and prepare a new query
3. Now when you run `npm run build-types` you get generated hook in `src/graphql/hooks.test.generated.ts`

### Screenshots



### Pull Request Checklist



1. [ ] This code contains UI changes
4. [ ] All visible strings are translated with proper context including data-formatting
5. [ ] Attributes `[data-test-id]` are added for new elements
6. [ ] Changes are mentioned in the changelog
7. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config



API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
MARKETPLACE_URL=https://apps.saleor.io
SALEOR_APPS_ENDPOINT=https://apps.saleor.io/api/saleor-apps

### Do you want to run more stable tests?
To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
8. [ ] productType
9. [ ] shipping
10. [ ] customer
11. [ ] permissions
12. [ ] menuNavigation
13. [ ] pages
14. [ ] sales
15. [ ] vouchers
16. [ ] homePage
17. [ ] login
18. [ ] orders
19. [ ] products
20. [ ] app
21. [ ] plugins
22. [ ] translations
23. [ ] navigation
24. [ ] variants
25. [ ] payments

CONTAINERS=1
